src/Nethermind/Nethermind.Merge.Plugin/PoSSwitcher.cs

```diff
@@ -155,13 +155,37 @@ public bool TryUpdateTerminalBlock(BlockHeader header)
             {
                 TerminalBlockReached?.Invoke(this, EventArgs.Empty);
                 _hasEverReachedTerminalDifficulty = true;
-                if (_logger.IsInfo) _logger.Info($"Reached terminal block {header}");
+                PrintPoSBanner(header);
```

Comment:
Printing banner in this place is wrong, maybe let's revert this change and only add the log to the codebase, without calling the method for now?

Reply:
Yes, here we just received terminal block for the first time (last PoW block), but there can be a few competing terminal blocks and PoS is starting when we receive first FCU (or NewPayload, not sure)

Reply:
Yeah terminal is POTENTIALLY last PoW block, but they can compete.